src/v2/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -78,3 +98,45 @@ const DefaultSuggestion: React.FC<SuggestionItemProps> = ({
     </>
   )
 }
+
+const QuickNavigation: React.FC<{
+  href: string
+  showArtworksButton: boolean
+  showAuctionResultsButton: boolean
+}> = ({ href, showArtworksButton, showAuctionResultsButton }) => {
+  if (!showArtworksButton && !showAuctionResultsButton) return null
+
+  return (
+    <Flex mt={1}>
+      {!!showArtworksButton && (
+        <QuickNavigationItem to={`${href}/works-for-sale`}>
+          <ArtworkIcon mr={0.5} />
+          Artworks
+        </QuickNavigationItem>
+      )}
+      {!!showAuctionResultsButton && (
+        <QuickNavigationItem to={`${href}/auction-results`}>
+          <AuctionIcon mr={0.5} />
+          Auction Results
+        </QuickNavigationItem>
+      )}
+    </Flex>
+  )
+}
+
+const QuickNavigationItem: React.FC<{ to: string }> = ({ children, to }) => {
+  const { router } = useRouter()
+
+  const onClick = event => {
+    event.preventDefault()
+
+    router ? router.push(to) : window.location.assign(to)
+  }
+  return (
+    <Clickable display="flex" onClick={onClick} as="div">
```

Comment:
This sort of negates the benefit of using a `Clickable` — I realize it's invalid to nest buttons like this. Really we'd want to either adjust the hit areas so as to be distinct or handle this using `role="button" tabIndex={0} onKeyPress={...}` etc. I'd leave it alone for now since I'm re-working a bunch of this, but more of a heads up.

Reply:
Thanks!  You're right, adjusting the hit area sounds cleaner. I used `onClick` as a workaround here to not nest buttons...